### DISCUSSION TITLE
elem_arg??

### DISCUSSION THREAD

**Discussion by joe61vette** [idaholab/moose#21300] [Q&A Modules: Navier-Stokes] [answered]
Hello:

In fvkernels, two forms are used for elem_arg in gatherRCData.  Namely:

const auto elem_arg = makeElemArg(&elem);

or

const auto & elem_arg = makeElemArg(&elem);

Is there any advantage to using this as a reference?

Thanks,
  Joe Kelly

**Comment by GiudGiud** [comment, marked as answer]
Hello

So when using a reference you avoid copying it, so it's technically more efficient. 
BUT it's unlikely to matter right now. We have other performance sinks to get to first. 

Guillaume
